packages/jasmine-enzyme/src/index.js

```diff
@@ -29,3 +29,13 @@ export default function jasmineEnzyme() : void {
     });
   });
 }
+
+// Also expose enzymeMatchers directly so that the matchers can be added on a per-spec basis
+// instead of globally on the jasmine object. This also supports older versions of jasmine where
+// jasmine.addMatchers isn't defined and matchers must be added to the spec in a beforeEach().
```

Comment:
jasmine 1.3.1. Yeah, it is necessary for us - it's probably going to be a while before we get our thousands of tests all moved over and onto a newer version of jasmine. I understand if you don't want to take this change; we could keep it just in our fork. Overall though I think it's nice to expose the matchers directly rather than just calling jasmine.addMatchers() internally - it gives consumers more control over when and how to register matchers.